### ISSUE TITLE
merge losing hits and order information

### ISSUE THREAD

**Issue by kannaiah** [SimonKagstrom/kcov#219]
Merged coverage report not showing hits and order information.
I build kcov from source `e42bdbbd3`

Also where can I find information on hits, possible hits and order (regards to what they mean).

**Comment by SimonKagstrom** [comment]
Yes, unfortunately number of hits and ordering isn't available in the merged reports. This is actually somewhat by design, since merge mode merges slightly incompatible modes. Python and Bash reports hit count per line, while compiled code reports if a block in a line has been hit - so there can be one or multiple hits per line.

I resorted to simply removing the hit count in the merge mode, since it made the implementation simpler. Adding hit count and ordering is doable, but probably a bit messy to implement.


As for the information, I know it's lacking. kcov has a man-page which describes hit count (but not ordering, sorry about that). man-pages are old-fashioned nowadays, so I think the best way would probably be to document it using markdown on github (linking to pages from the README.md), or what would you say?

Anyway:

- Possible hits are the number of instrumentation points for a particular line of code
- Hits are the number of instrumentation points for a line which were executed
- Order is the order of instrumentation point hits, within a file (typically not very interesting)

So for example
```
if (cond_a() || cond_b())
   ...
```

might generate two instrumentation points, one for each part of the conditional. Or not, it's difficult to know what the compiler makes of the code.

**Comment by kannaiah** [comment]
Thank you. Now it makes much more sense.

About merging I have gone through previous issues filed but unable to figure out if I can merge more than two reports into one. I'm trying to generate separate report for different set of tests and combine them I have more than few test sets.

**Comment by SimonKagstrom** [comment]
Yes, you can run with ```--merge```, like this:

```
kcov --merge /tmp/out /tmp/report1 /tmp/report2
```
